### ISSUE TITLE
Enemies don't die from magic/melee

### ISSUE THREAD

**Issue by Soveringg** [tiltedphoques/TiltedEvolution#228]
When you try to kill an enemy with magic or weapons, his health drops to zero, But he does not die and continues to attack. Sometimes it goes away by itself(just attack for a long time), but sometimes he can only be killed with a finishing blow.

**Comment by Neusor** [comment]
Not enough info for now, possibly related to #241 
Try this as a possible workaround: https://www.nexusmods.com/skyrimspecialedition/mods/13395

**Comment by Fabricio20** [comment]
I was able to reproduce this issue twice, but it's not an issue that happens always. It doesn't seem to happen with normal melee weapons but does happen more frequently with spells. When looking at the entity inspector, I noticed that the Health (H?) field was at a negative number (and every hit made it bigger), but IsDead was = 0.

NPCs in this particular state will not die from normal attacks or spells, they can only be killed with a Finishing Blow, additionally, if you keep hitting the NPC it's as if the NPC has died multiple times, so when they actually die, the loot drop is multiplied to about the amount of times you had "killed" the NPC.

Leaving the area / walking away from the bugged NPC will cause them to drop dead after a few meters, possibly near Skyrim's "npc will stop chasing you" distance.
